config/environments/development.rb

```diff
@@ -35,13 +35,53 @@
     config.cache_store = :null_store
   end
 
+  ## Begin - Active storage configuration
   # Store uploaded files on the local file system (see config/storage.yml for options).
   config.active_storage.service = :local
+  ## End - Active storage configuration
+
+  ## Begin - Action mailer configuration
+  if ENV['SMTP_SERVER'].present?
+    config.action_mailer.perform_deliveries = true # Enable email delivery.
+    ## Begin - SMTP configuration
+    config.action_mailer.delivery_method = :smtp
+    config.action_mailer.smtp_settings = {
+      # For more information: https://guides.rubyonrails.org/action_mailer_basics.html#action-mailer-configuration.
+
+      # SMTP server address i.e. smtp.gmail.com.
+      address: ENV.fetch('SMTP_SERVER', nil),
+      # SMTP PORT used by the SMTP server i.e. 25.
+      port: ENV.fetch('SMTP_PORT', nil),
+      # SMTP domain for HELO message, i.e. gmail.com
+      domain: ENV.fetch('SMTP_DOMAIN', nil),
+      # SMTP User username.
+      user_name: ENV.fetch('SMTP_USERNAME', nil),
+      # SMTP User password.
+      password: ENV.fetch('SMTP_PASSWORD', nil),
+      # SMTP Authentication mode ('plain', 'login', 'cram_md5'), when set to 'plain' the credentials will be sent in plaintext.
+      authentication: ENV.fetch('SMTP_AUTH', nil),
+      # SMTP STATTLS negociation, NOTE: this will not fail when STARTTLS is not supported by the SMTP server.
+      enable_starttls_auto: ENV.fetch('SMTP_STARTTLS_AUTO', true), # By default, STARTTLS will be preferred but not required.
+      # SMTP STARTTLS, when enabled email delivery will fail for SMTP servers that unsupports STARTTLS.
+      enable_starttls: ENV.fetch('SMTP_STARTTLS', false), # By default, STARTTLS support will not be mandatory.
+      # SMTP over secure TLS connection (SMTPS).
+      tls: ENV.fetch('SMTP_TLS', 'false') != 'false', # By default, SMTPS is not used.
+      # OpenSSL SSL certificate verification for TLS connections ('none', 'peer').
+      # By default, SSL validations should pass.
+      openssl_verify_mode: ENV.fetch('SMTP_SSL_VERIFY', 'true') == 'false' ? OpenSSL::SSL::VERIFY_NONE : OpenSSL::SSL::VERIFY_PEER
```

Comment:
There is way too many comments here. There should only be comments for code that is complex. 

In this case, we can probably remove all the comments